mtp_cashbook/templates/cashbook/change_notification.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load mtp_common %}
+{% load staticfiles %}
+
+{% block body_classes %}mtp-change-notification{% endblock %}
+
+{% block inner_content %}
+
+  <header>
+    {% language_switch %}
+    <h1 class="heading-large">{% trans 'Digital cashbook is now connected to NOMIS' %}</h1>
+  </header>
+
+  <div class="mtp-change-notification__page">
+    <img class="mtp-change-notification__graphic" src="{% static 'images/nomis-change-graphic.png' %}"/>
+    <img class="mtp-change-notification__graphic2x" src="{% static 'images/nomis-change-graphic@2x.png' %}"/>
```

Comment:
would be simpler to use a `srcset` attribute on one image element and avoid having to do any styles. like in the training section